lib/pdk/config.rb

```diff
@@ -3,24 +3,29 @@
 module PDK
   class Config
     autoload :JSON, 'pdk/config/json'
-    autoload :JSONSchemaNamespace, 'pdk/config/json_schema_namespace'
-    autoload :JSONSchemaSetting, 'pdk/config/json_schema_setting'
```

Comment:
I would suggest keeping these autoloads in place rather than adding the top level requires as done below

Reply:
+1. Will do

Reply:
Done.